### ISSUE TITLE
Male and female populations at the province level are identical for each province

### ISSUE THREAD

**Issue by cliftonmcintosh** [CodeforNepal/nepalmap_federal#43]
Prior to recent updates of more data added by @pratimakandel , the male and female populations for each province are identical. Please see 
https://github.com/Code4Nepal/nepalmap_federal/blob/7bc04d17ca409f722af09d939028159e5d86d426/sql/population.sql

That is the commit before @pratimakandel began adding data for missing districts.

There must be some sort of error in the calculation.

**Comment by cliftonmcintosh** [comment]
SQL to get the province population figures from the district totals
```
select concat(pg.geo_code, ',province,', p.sex, ',', sum(p.total)) from wazimap_geography pg, wazimap_geography g, population p where p.geo_level = 'district' and p.geo_code = g.geo_code and pg.geo_code = g.parent_code and g.geo_level = 'district' and pg.geo_level = 'province' group by pg.geo_code, p.sex order by pg.geo_code desc, p.sex desc;
          concat           
---------------------------
 7,province,male,1124864
 7,province,female,1258383
 6,province,male,737483
 6,province,female,784244
 5,province,male,1917059
 5,province,female,2147564
 4,province,male,1060103
 4,province,female,1295533
 3,province,male,2509586
 3,province,female,2600631
 2,province,male,2694747
 2,province,female,2674024
 1,province,male,2069237
 1,province,female,2290138
```

**Comment by cliftonmcintosh** [comment]
#44 has been submitted to address this issue.

**Comment by cliftonmcintosh** [comment]
Corrected with merge of #44
